pithos/plugins/notify.py

```diff
@@ -45,6 +45,14 @@ def on_prepare(self):
         if 'action-icons' in caps:
             self.notification.set_hint('action-icons', GLib.Variant('b', True))
 
+        # GNOME Shell 3.20 or higher has bultin MPRIS functionality that makes
+        # persistent song notifications redundant.
+        min_version = Gtk.get_major_version() >= 3 and Gtk.get_minor_version() >= 20
```

Comment:
I see what you're getting at though. Gtk version would get the version of the bundled Gtk and the server version would get you the Gtk/shell version of the host